internal/v1/api.yaml

```diff
@@ -1293,6 +1314,23 @@ components:
           type: string
         signature:
           type: string
+    RecommendPackageRequest:
+      type: object
+      properties:
+        packages:
+          type: array
+          items:
+            type: string
+        recommendedPackages:
+          type: integer
+          format: int32
+    RecommendationsResponse:
+      type: object
+      properties:
+        packages:
+          type: array
+          items:
+            type: string
```

Comment:
Please add required flag to all fields for the same reason - the caller of this service does not need to fight with pointers, this must return something (e.g. an empty array).

Reply:
Response object has `packages` element which is not required.